### DISCUSSION TITLE
How to add drop shadow behind InfoText?

### DISCUSSION THREAD

**Discussion by Xio1996** [Stellarium/stellarium#4012] [Q&A] [answered]
Hi,

How do you turn on the info text drop shadow?

Couldn't find a setting in the GUI and tried adding a flag to config.ini, with no success.

[gui]
flag_info_shadow                          = true

_List of changes in version 24.3.351 / 24.4RC2 / 24.3-3c8d3c4 (2024-12-16)
    Added optional drop shadow behind InfoText_

Many thanks

Pete

**Comment by gzotti** [comment, marked as answer]
Yes, like that. The effect is subtle (just to emphasize without becoming too noticeable), but it's there.
